tests/test_butler.py

```diff
@@ -159,6 +159,33 @@ def testMakeRepo(self):
         self.assertIn("datastore.formatters", full)
         self.assertNotIn("datastore.formatters", limited)
 
+    def testExposureCompositePutGet(self):
```

Comment:
This test needs to skip if afw does not exist. (I deliberately kept Exposure out of this file) We need to be clear about our optional formatters. The datastore tests are explicitly isolated for optional afw dependency.